CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+
+**Threats:** threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.
+
+### Unwelcoming behavior
+
+#### Blatant acts and micro-aggressions
+Blatant micro-aggressions involve making statements that assume 
+negative characteristics in a blanket fashion because of someone's identification 
+with a particular group. We take micro-aggressions seriously,  especially 
+important when they target traditionally marginalized groups (e.g., 
+explicitly racist, sexist, homophobic, or transphobic statements).
+
+ > For example, arguing that some people are less intelligent because 
+ > of their gender, race, religion. 
+ 
+#### Subtle and unintentional micro-aggressions 
+Subtle acts of aggression refer to small, unintentionally harmful
+mistakes made in conversation. These are not necessarily a code of conduct 
+violation when made by mistake. 
+
+However, repeating something after it has 
+been pointed out to you that you broke a social rule, or antagonizing or 
+arguing with someone who has pointed out your micro-aggression is 
+considered unwelcoming behavior, and is not allowed in the pyOpenSci community.
+
+#### Maliciousness towards other community members
+Maliciousness towards other community members: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong at pyOpenSci. If somebody makes such a statement without malice, they may still be in violation of the Code of Conduct if their actions are deemed especially/repeatedly unpleasant (see below).
+
+Being especially or repeatedly unpleasant: for example, if we’ve received reports from multiple pyOpenSci users, team members, or collaborators of agitating, rude, or especially distracting behavior over an extended period of time.
+
+
+## Scope in which the pyOpenSci code of conduct is applied
+
+pyOpenSci community members are held to the standards outlined in this 
+code of conduct when interacting in the pyOpenSci Slack or GitHub 
+repositories, when interacting in-person at events where they could 
+represent pyOpenSci (this is most professional events), in physical spaces 
+with other pyOpenSci team members or collaborators, or in any Community 
+Initiatives (our fiscal sponsor) space.
+
+In addition, the pyOpenSci community and experience often extends outside 
+those spaces—pyOpenSci community members may go on walks together to get 
+lunch, attend meetups or conferences as a group, communicate on social 
+media, or interact with each other in other communities. 
+
+Abusive or unwelcoming behavior between community members still has a 
+profound impact on individuals and on the community when it happens beyond 
+our walls. The pyOpenSci Executive Director and the Advisory Council will 
+use our discretion when deciding whether to enforce this code of conduct 
```

Comment:
"use our discretion in deciding whether to enforce this code" sounds like we might decide to ignore bad behaviour

Reply:
i hear you. how would you reword? i think what is hard here is that if the behavior happens away from our organizations activities we'd have to figure out whether it was in violation or not and whether it happened as each party describes. 

Very open to word changes here that send a better message but still allow us to "Decide" in case it's outside of our jurisdiction?